internal/context/ngap_handler.go

```diff
@@ -254,15 +254,17 @@ func HandleHandoverRequestAcknowledgeTransfer(b []byte, ctx *SMContext) (err err
 
 		var indirectFowardingPDR *PDR
 
-		if pdr, err := ANUPF.AddPDR(); err != nil {
+		if pdr, errAddPDR := ANUPF.AddPDR(); errAddPDR != nil {
+			err = errAddPDR
 			return err
```

Comment:
Remove `err` at function declaration.

And return `errAddPDR` directly. So does the `errGenerateTEID`